examples/Perl/googlevoice.pl

```diff
@@ -0,0 +1,202 @@
+#!/usr/bin/perl
+#
+# Uses snowboy to wake up and then collect audio to send to Google Speech API for further recognition
+# Takes Personal Model as a first argument
+
+use lib 'Snowboy';
+
+use Audio::PortAudio;
+use Data::Dumper;
+use Math::Round qw/round/;
+use LWP::UserAgent;
+use JSON;
+use Statistics::Basic qw(:all);
+use Time::HiRes qw(gettimeofday tv_interval);
+use IO::Handle;
+
+use Snowboy;
+
+die "Usage: $0 <model.pmdl>" 
+  if scalar @ARGV < 1;
+
+STDOUT -> autoflush (1);
+binmode STDOUT, ':utf8';
+
+# Your choice of language
+use constant LANG => 'en';
+
+# Audio format
+use constant RATE => 16000;
+use constant NUMCHANNELS   => 1;
+use constant BITSPERSAMPLE => 16;
+
+# Samples per data chunk count
+use constant SAMPLES => 640;
+
+# Detect 500ms silence (12 blocks * 40 ms = after 500ms of speech)
+use constant TRAILING_SILENCE_BLOCKS => 12;
+
+$models = '/var/spool/models';
```

Comment:
Let's remove the variable `models`:

1. We assume the user will provide the full path with the model, so `$models/$ARGV[0]` below can simply be `$ARGV[0]`

2. We assume user will run it from the `examples/Perl` directory, so we can use `resources/common.res` instead of `$models/common.res`